src/main.cpp

```diff
@@ -54,6 +54,7 @@ using namespace std;
 
 CCriticalSection cs_main;
 
+uint64_t nGlobalPruneAfterHeight = 100000;
```

Comment:
Please fix doc of https://github.com/jtimon/bitcoin/blob/7992d97fa4e92351f9f4fa155384ea2181b28f56/src/main.h#L218